### ISSUE TITLE
Can't install the funnel dependency

### ISSUE THREAD

**Issue by nek555** [MartinPacker/md2pptx#139]
md2pptx has a dependency on funnel.

When I try to install funnel, I get an error message.

python -m pip install funnel
Collecting funnel
  Using cached Funnel-0.1.tar.gz (1.4 kB)
  Preparing metadata (setup.py) ... done
Collecting Flask>=0.8
  Using cached Flask-2.2.3-py3-none-any.whl (101 kB)
Collecting Beaker==1.6.3
  Using cached Beaker-1.6.3.tar.gz (52 kB)
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error

  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [1 lines of output]
      error in Beaker setup command: use_2to3 is invalid.
      [end of output]

I'm working in a Python 3.10.6  venv on Debian 11 linux (within Window WSL). 

Is there a workaround for this?

**Comment by MartinPacker** [comment]
Hello Norbert.

Funnel is built in to md2pptx - as a separate file in the repo. No separate (pip) installation required (or, as you found out, possible). So md2pptx should run.

I'll take this issue as a chance to consider improving the documentation.

**Comment by MartinPacker** [comment]
See updated README.md. Committed and pushed. Happy to make further edits. Might need also to update User Guide.

**Comment by nek555** [comment]
Hello Martin,
Thanks for the quick response.  I see where I went wrong.  I did not clone the entire repo when I did my installation.  I just did this
python -m pip install python-pptx
and then copied down the python script md2pptx and ran it.  Of course I got the error about funnel not being available, and then tried to install funnel using pip. (normal procedure when using python)

For anyone else installing this:  clone the entire repo.  Install python-pptx with pip. md2pptx will use the version of funnel that is in the repo.

Once I did that, everything worked beautifully. I was able to convert the md files in the test folder.

This is powerful and useful software, and the author responds quickly to reported issues.

Thanks again.
Norbert Kremer

**Comment by nek555** [comment]
Hello Martin,
I added a comment to the issue and closed it.  I did not realize that I 
needed to clone the entire repo.  Once I did that, everything worked 
like a charm.

I really appreciate your quick response.

My ultimate goal here is to go from markdown to google slides. There is 
another project on github that does this in one step, using some API's 
in Google Cloud.   That other project appears to be dead, a lot of 
people are having trouble getting to work and the original author is 
nowhere to be found.  I wrestled with it for 3 hours and then gave up.  
Here is the repo for that other project 
https://github.com/googleworkspace/md2googleslides

So, at least now I have a path to move ahead.   I can use your program 
to go from md to pptx, and then Google will import pptx and convert to 
google slides.

Thanks again,
Norbert Kremer
***@***.***
